frontend/src/components/edit-widget-modal.js

```diff
@@ -153,7 +153,7 @@ export class EditWidgetModal extends React.Component {
               </HelperText>
             </FormHelperText>
           </FormGroup>
-          {componentLoaded ? widgetType.params.map(param => {
+          {componentLoaded ? widgetType?.params.map(param => {
```

Comment:
if widgetType doesn't exist, you will return the string "undefined" in the dom.

I'd suggest (widgetType?.params||[]).map

Or 

componentLoaded && !!widgetType ?

Reply:
thanks!